scripts/crawler/tbc_web_crawler/settings.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
```

Comment:
Is it possible to remove the unused settings? You could also provide a link to the actual settings template in the original scrapy source code